viscm/gui.py

```diff
@@ -871,27 +954,33 @@ def loadpyfile(path):
     is_native = True
     cmtype = "linear"
     method = "Bezier"
-    ns = {'__name__': '',
-          '__file__': os.path.basename(path),
-          }
-    with open(args.colormap) as f:
-        code = compile(f.read(),
-                       os.path.basename(args.colormap),
-                       'exec')
+    ns = {
+        "__name__": "",
+        "__file__": os.path.basename(path),
+    }
+
+    # FIXME: Should be `args.colormap` should be `path`?
+    with open(args.colormap) as f:  # noqa: F821
+        code = compile(
+            f.read(),
+            os.path.basename(args.colormap),  # noqa: F821
+            "exec",
+        )
```

Comment:
The `FIXME` above and `noqa` comments document a bug found by `ruff`. Will get back to this when I'm ready to dig into the python loading behavior and build a unit test :)